### ISSUE TITLE
Reversed direction in GMTgrid, v0.15.0

### ISSUE THREAD

**Issue by hydrocoast** [GenericMappingTools/GMT.jl#345]
Hi, 

I found that functions that create `GMT.GMTgrid` (e.g. `surface`) of v0.15.0 show different illustration from those of v0.14.1.

For example, the following codes generate a figure:
```julia
using GMT

X = 0:100
Y = 0:50
Z = [sqrt(x^2+y^2) for y in Y, x in X]

G = surface([repeat(X,inner=(51,1)) repeat(Y,outer=(101,1)) vec(Z)], I=1, R="0/100/0/50")

cpt = makecpt(C=:rainbow, D=true, T="0/100")
grdimage(G, J="X10/5", R="0/100/0/50", C=cpt, Q=true, show=true)
```

Using v0.14.1, the output looks like:
![v0 14 1](https://user-images.githubusercontent.com/27009260/75601253-eba5ee00-5afc-11ea-8c5f-ae5b5204a294.png)

Using v0.15.0, the direction of the vertical axis is reversed:
![v0 15 0](https://user-images.githubusercontent.com/27009260/75601275-ffe9eb00-5afc-11ea-99e9-16da37bb784c.png)

I think the behavior of v0.14.1 is proper, so I am looking for what part of modification from v0.14.1 to v0.15.0 causes the difference.

The environment is as follows:
  OS: Ubuntu 18.04 LTS
  Julia version: v1.3.1
  GMT version: 6.0.0

Thank you,

**Comment by joa-quim** [comment]
@hydrocoast  Thank you for catching this. It was a bug in the commit that replaced *global* by *const global*. Fixed and will tag a 0.16.0 soon.

And, BTW, to quick visualize a grid ``G`` you can just do ``imshow(G)``

**Comment by hydrocoast** [comment]
Thank you for the quick response and update.
